README.md

```
@@ -99,6 +99,111 @@ result = module.process()
 print(result)
 ```
 
+## Local Development with Docker
+
+The full stack runs with Docker Compose, including the FastAPI gateway, React frontend, PostgreSQL, Redis, and RQ worker.
+
+### Prerequisites
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)